client/my-sites/woocommerce/dashboard/required-plugins-install-view.jsx

```diff
@@ -597,7 +598,8 @@ class RequiredPluginsInstallView extends Component {
 		const { engineState, progress, totalSeconds } = this.state;
 
 		if ( ! hasPendingAT && 'CONFIRMING' === engineState ) {
-			return this.renderConfirmScreen();
+			return <WoopLandingPage startSetup={ this.startSetup } />;
+			//return this.renderConfirmScreen();
```

Comment:
```suggestion
```

Reply:
Should also delete the function.

Reply:
Thanks for catching that, it reappeared from a merge conflict 😂